index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>freeCodeCamp Survey Form</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <h2 id="title">freeCodeCamp Survey Form</h2>
+    <p class="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
```

Comment:
- [ ] I think the parent tag here should be a div tag, not a "p" tag. The "p" tag defines a paragraph while the div tag is simply an HTML container. Its only purpose is to let you group together other document elements to which to apply the same CSS style